CMakeLists.txt

```diff
@@ -0,0 +1,366 @@
+cmake_minimum_required(VERSION 3.14)
+
+project(KTX-Software)
+
+include(CTest)
+
+if(NOT ANDROID)
+    find_package(OpenGL)
+endif()
+
+if(NOT IOS AND NOT ANDROID)
+    find_package(Vulkan)
+endif()
+
+if(IOS)
+    set(CMAKE_OSX_DEPLOYMENT_TARGET "10.0" CACHE STRING "Minimum iOS deployment version")
+endif()
```

Comment:
The equivalent Xcode setting is called `IPHONEOS_DEPLOYMENT_TARGET`. I hope this is a typo here and not a stupid design decision in CMake.

There also needs to be an

```
if (macOS)
    set(CMAKE_OSX_DEPLOYMENT_TARGET "10.11" CACHE STRING "Minimum macOS deployment version")
endif()
```

And please change "version" to "target" in those strings.

Reply:
Well, `CMAKE_OSX_DEPLOYMENT_TARGET` can be used to set the deployment target for any Apple platforms (including macOS, iOS, watchOS, etc.).
I'll make sure there's a distinct default deployment target for both macOS and iOS.